src/Templates/Boilerplate/Bit.Boilerplate/src/Client/Boilerplate.Client.Web/wwwroot/service-worker.published.js

```diff
@@ -23,7 +23,18 @@ self.addEventListener('notificationclick', function (event) {
     const pageUrl = event.notification.data.pageUrl;
     if (pageUrl != null) {
         event.waitUntil(
-            clients.openWindow(pageUrl)
+            clients
+                .matchAll({
+                    type: 'window',
+                    includeUncontrolled: true,
+                })
+                .then((clientList) => {
+                    for (const client of clientList) {
+                        client.postMessage({ key: 'PUBLISH_MESSAGE', message: 'NAVIGATE_TO', payload: pageUrl });
+                        return client.focus();
```

Comment:
same here:

`client.postMessage && client.postMessage.(...)`

and

`client.focus && client.focus()`